docs/reference/query-dsl/percolate-query.asciidoc

```diff
@@ -144,6 +144,41 @@ In that case the `document` parameter can be substituted with the following para
 `preference`:: Optionally, preference to be used to fetch document to percolate.
 `version`:: Optionally, the expected version of the document to be fetched.
 
+[float]
+==== Percolating in a filter context
+
+In case that you are not interested in the score it is beneficial to wrap the
```

Comment:
maybe be explicit about the fact it could make things faster, eg.

> In case you are not interested in the score, better performance can be expected by wrapping
> the percolator query in a `bool` query's filter clause or in a `constant_score` query: